lib/Core/GPS/src/processUbloxGPS.c

```diff
@@ -493,6 +498,22 @@ void decodeNavPvt(char *msg, GpsData_t *GPSData)
     GPSData->updateFlagForEachCall |= 1 << GOT_VTG_MSG;
 }
 
+void extractHeadingFromRelPosNed(char *msg, GpsData_t* GPSData) {
+    int relPosHeading = *( (int *) msg + 24 );
+    GPSData->relPosHeading = relPosHeading * 1e-5;
+
+    unsigned int accRelPosHeading = *( (unsigned int*) msg + 52 );
+    GPSData->accRelPosHeading = accRelPosHeading * 1e-5;
+
+    // Here I don't really know if these are the only flags I should look for.
+    // For now it is just a guess of mine
+    unsigned int flags = * ( (unsigned int*) msg + 60);
+    bool gnssFixOK = flags & 0x01;
+    bool isMoving = (flags >> 5) & 0x01;
```

Comment:
You can safely ignore this one, but should verify that carrSoln == 2 (RTK fixed solution)

Reply:
Which on "this" refers to? gnssFixOK or isMoving?

Reply:
isMoving